mixins/focus-visible-polyfill-mixin.js

```diff
@@ -14,7 +14,10 @@ export const FocusVisiblePolyfillMixin = superclass => class extends superclass
 		}
 
 		window.addEventListener('focus-visible-polyfill-ready', () => {
-			window.applyFocusVisiblePolyfill(this.shadowRoot);
```

Comment:
No harm in doing this check, but it certainly seems strange to need this, since the function is attached to the window before the event is dispatched. (https://github.com/WICG/focus-visible/blob/master/src/focus-visible.js#L283)

Reply:
Agreed. After this goes in, I'm going to monitor the JS errors closely and make sure it actually does solve the issue. All I can think of is that it's some kind of timing issue... maybe as the page is unloading the event is firing. I have no idea.

Reply:
Actually... looking at when the errors occurred, I think this was some kind of automation that triggered it. It happened several hundred times all at the exact same second the morning of Nov 2 and then the same thing Oct 30.